src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/DefendantResponseContentProvider.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.cmc.claimstore.documents.content;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.config.properties.notifications.NotificationsProperties;
+import uk.gov.hmcts.cmc.claimstore.documents.ClaimDataContentProvider;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.FullDefenceResponse;
+import uk.gov.hmcts.cmc.domain.models.Response;
+import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
+
+import java.util.HashMap;
+
+import java.util.Map;
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatDateTime;
+
+@Component
+public class DefendantResponseContentProvider {
+
+    private final PartyDetailsContentProvider partyDetailsContentProvider;
+    private final ClaimDataContentProvider claimDataContentProvider;
+    private final NotificationsProperties notificationsProperties;
+
+    public DefendantResponseContentProvider(
+        final PartyDetailsContentProvider partyDetailsContentProvider,
+        final ClaimDataContentProvider claimDataContentProvider,
+        final NotificationsProperties notificationsProperties) {
```

Comment:
would be nice to move `) {` down to the next line, it reduces diffs when new parameters are added and is easier to read IMO, the rule I go by is >= 3 parameters and split them per line
REC

Reply:
@timja-kainos It wouldn't reduce diffs, because you'd still have to add a comma after last parameter when adding a new one, but I agree about it being easier to read.